pyfolio/tests/test_timeseries.py

```diff
@@ -311,12 +311,13 @@ class TestStats(TestCase):
         (simple_month_rets, utils.MONTHLY, 0.006885932704891129)
     ])
     def test_annual_ret(self, returns, period, expected):
-        self.assertEqual(
+        assert_almost_equal(
```

Comment:
Yup, I get the logic of why we want to use `assert_almost_equal` in the general case (even 0.1 + 0.2 has floating-point issues). What I meant to ask was why it changed between this branch and master - was the test failing on your computer on master as well?